onebusaway-transit-data-federation/src/main/java/org/onebusaway/transit_data_federation/impl/ArrivalAndDepartureServiceImpl.java

```diff
@@ -682,10 +661,12 @@ private boolean setPredictedTimesFromTimepointPredictionRecords(
           instance.getStop().getId());
       boolean sequenceMatches = tpr.getStopSequence() > 0
           && tpr.getStopSequence() == gtfsSequence;
-
       if (!tripMatches || !stopMatches)
         continue;
-
+      if (tpr.isSkipped()) {
+        _log.debug("Skipped Stop: " + instance.getStop().getId().getId());
+        break;
```

Comment:
@NJBL @sheldonabrown I just called for a vote on https://github.com/google/transit/pull/139, which is an official proposal for the GTFS-realtime spec that clarifies that delays should be propagated over SKIPPED stops, which is Google's implementation.  It's been a long time since I've dived into this code in OBA, but I think this may need to be changed to a `continue` instead of `break` (and counters properly incremented) to propagate delays past the SKIPPED stop?  IIRC this `break` would result in scheduled times showing up after the `SKIPPED` stop (i.e., no real-time info).

Reply:
@NJBL @sheldonabrown https://github.com/google/transit/pull/139 is now approved and part of the GTFS-rt spec.  Can you please confirm whether OBA follows this behavior or not?